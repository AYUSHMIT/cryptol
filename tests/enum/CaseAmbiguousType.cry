f e =
  case e of
    _ -> ()
