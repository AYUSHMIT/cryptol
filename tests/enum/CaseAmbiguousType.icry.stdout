Loading module Cryptol
Loading module Cryptol
Loading module Main

[error] at CaseAmbiguousType.cry:2:3--5:0:
  Type mismatch:
    Expected an enum type, but
    the inferred type (?a) is ambiguous.
    Try giving the expression an explicit type.
    When checking cased expression
  where
  ?a is the type of 'e' at CaseAmbiguousType.cry:1:3--1:4
